lisa/executable.py

```diff
@@ -164,6 +164,10 @@ def create(cls, node: Node, *args: Any, **kwargs: Any) -> Tool:
             windows_tool = cls._windows_tool()
             if windows_tool:
                 tool_cls = windows_tool
+        if "FreeBSD" in node.os.name:
```

Comment:
use `elif`, so the logic is clearer, and avoid strange problems.

Reply:
update